libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs

```diff
@@ -393,12 +399,12 @@ public async Task DeleteConversationMemberAsync(ITurnContext turnContext, string
         {
             if (turnContext.Activity.Conversation == null)
             {
-                throw new ArgumentNullException("BotFrameworkAdapter.deleteConversationMember(): missing conversation");
+                throw new NullReferenceException("BotFrameworkAdapter.deleteConversationMember(): missing conversation");
             }
```

Comment:
Please don't change this. That's changing behaviorial semantics and shouldn't be done here.

Reply:
Reverted